tests/integration/suite/placement/table/vnodes_api_level_10.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package table
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	placementv1pb "github.com/dapr/dapr/pkg/proto/placement/v1"
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/placement"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(vNodesAPILevel10))
+}
+
+type vNodesAPILevel10 struct {
+	place *placement.Placement
+}
+
+func (v *vNodesAPILevel10) Setup(t *testing.T) []framework.Option {
+	v.place = placement.New(t,
+		placement.WithLogLevel("debug"),
```

Comment:
This makes placement chatty and therefore increases the buffer we write to in-memory for our runners. Is this really necessary?

Reply:
I don't think so. If needed, we can only add it locally, when debugging.
I'll remove.